crates/uv/tests/it/init.rs

```diff
@@ -1207,10 +1207,16 @@ fn init_workspace() -> Result<()> {
         name = "project"
         version = "0.1.0"
         requires-python = ">=3.12"
-        dependencies = ["anyio==3.7.0"]
+        dependencies = [
+            "anyio==3.7.0",
+            "foo",
```

Comment:
I wonder if we should have an opt-out for this? It seems a little awkward since there's no version bound. What if you wanted it as a dev dependency? I wonder if we should even do this by default for these reasons?

Reply:
In the issue I suggested a toggle and @charliermarsh disagreed, so perhaps we should revisit that conversation briefly?

Reply:
(it seems more annoying to _undo_ this automatic add then to need t do it manually?)